CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Bug Fixes
 
 * Add new `msgfees` `NhashPerUsdMil`  default param to param space store on upgrade (PR [#875](https://github.com/provenance-io/provenance/issues/875))
+* Marker module `authzHandler` function is double decrementing [#903](https://github.com/provenance-io/provenance/issues/903)
```

Comment:
This is probably a bit too deep for the changelog. Also, it's not actually double decrementing due some fortuitous  interplay involving when the `authzHandler` is called.

A better changlog entry might be along the lines of "Fix the `MarkerTransferAuthorization` Accept function and `TransferCoin` authz handling to prevent problems when other authorization types are used."

I did verify with a user of that authorization that it's not actually double decrementing. But if that function were to be used in other places, it'd probably end up double decrementing.

Reply:
Also, a `State Machine Breaking` entry should be added to the changelog for this since handling of authorizations other than `MarkerTransferAuthorization` is changing (getting fixed).

Reply:
Fixed.